src/Simplex_tree/concept/FiltrationValue.h

```diff
@@ -5,19 +5,67 @@
  *    Copyright (C) 2014 Inria
  *
  *    Modification(s):
+ *      - 2024/08 Hannah Schreiber: Update of the concept after several additions to the Simplex tree.
  *      - YYYY/MM Author: Description of the modification
  */
 
 /** \brief Value type for a filtration function on a cell complex.
-  *
-  * A <EM>filtration</EM> of a cell complex (see FilteredComplex) is
-  * a function \f$f:\mathbf{K} \rightarrow \mathbb{R}\f$ satisfying \f$f(\tau)\leq 
-  * f(\sigma)\f$ whenever \f$\tau \subseteq \sigma\f$. Ordering the simplices 
-  * by increasing filtration values (breaking ties so as a simplex appears after 
-  * its subsimplices of same filtration value) provides an indexing scheme 
-  * (see IndexingTag).
-  */
-  struct FiltrationValue {
-    /** \brief Operator < is a StrictWeakOrdering. */
-    bool operator<(FiltrationValue f1, FiltrationValue f2);
-  };
+ *
```

Comment:
We need to modify SimplexTreeOptions concept, cf.
https://github.com/GUDHI/gudhi-devel/blob/2a89fd56885ccabde60044816c4e8e0d1613b2cc/src/Simplex_tree/concept/SimplexTreeOptions.h#L22-L23

Reply:
I updated it. As the new interface is a bit more complicated, I just referenced the concept. Is it fine?

Reply:
I expect that's what was originally intended, otherwise I don't know if the concept was used anywhere.